resources/roles_edit.html

```diff
@@ -32,22 +32,38 @@
         newRow.find("input:visible").first().focus();
       };
 
+      var snooze = function(event) {
+        var duration = parseInt( $("input.snooze-duration").attr("value") );
+        var roleId = $("input#role-id").attr("value");
+        $.post("/api/v1/roles/" + roleId + "/snooze", {"duration": duration}, function(data) {
+          if (data["msg"]) {
+            $("span#snooze-until").text(data["msg"]);
+            $("span#snooze-until").removeClass("hidden");
+          }
+          $("#flash-message").html("Role snoozed");
+        });
+      }
 
       $("#roles-edit-page").on("click", "a.add", addRow);
 
       $("#roles-edit-page").on("click", "a.remove", removeRow);
+
+      $("#roles-edit-page").on("click", "button#snooze", snooze);
     });
   </script>
 
+  <input id="role-id" type="hidden" name="id" />
   <div id="flash-message">
     Your changes have been saved.
   </div>
 
-  <form method="POST">
+  <form class="update" method="POST">
 
     <div>
       <label for="name">Role name</label>
-      <input type="text" name="name" /><br/>
+      <input type="text" class="name" name="name" />
```

Comment:
Why do we need to add a class to this element? Doesn't the `name` attribute uniquely identify it?